presto-spi/src/main/java/io/prestosql/spi/type/SqlTimeWithTimeZone.java

```diff
@@ -67,7 +67,7 @@ public boolean equals(Object o)
         }
 
         SqlTimeWithTimeZone other = (SqlTimeWithTimeZone) o;
-        return precision == other.precision && normalizePicos(picos, offsetMinutes) == normalizePicos(other.picos, offsetMinutes);
+        return precision == other.precision && normalizePicos(picos, offsetMinutes) == normalizePicos(other.picos, other.offsetMinutes);
```

Comment:
cc @martint

Reply:
Good catch!

Reply:
Actually i think we should remove the normalize from here and compare both: picos, offsetMinutes by value.

I think we don't have to carry SQL semantics over to this class.
Also, we don't do this in `io.prestosql.spi.type.SqlTimestampWithTimeZone#equals` either